addons/mail/static/src/discuss/core/web/discuss_sidebar_categories.xml

```diff
@@ -12,17 +12,17 @@
 
     <t t-name="mail.DiscussSidebarCategory">
         <div class="o-mail-DiscussSidebarCategory d-flex align-items-center my-1" t-att-class="category.extraClass">
-            <div t-attf-class="flex-grow-1 d-flex align-items-baseline mx-1 btn p-0 text-start opacity-100-hover opacity-75" t-on-click="() => this.toggleCategory(category)">
+            <div t-attf-class="flex-grow-1 d-flex align-items-baseline mx-1 btn p-0 pt-2 text-start opacity-100-hover opacity-75" t-on-click="() => this.toggleCategory(category)">
                 <span t-if="store.channels.status === 'fetching'" class="o-visible-short-delay"><i class="o-mail-DiscussSidebarCategory-icon o-smaller me-1 fa fa-fw fa-circle-o-notch fa-spin opacity-50"/></span>
                 <i t-else="" class="o-mail-DiscussSidebarCategory-icon o-smaller me-1" t-att-class="category.open ? 'oi oi-chevron-down' : 'oi oi-chevron-right'"/>
                 <span class="btn-sm p-0 text-uppercase text-break fw-bolder o-smaller"><t t-esc="category.name"/></span>
             </div>
             <div class="d-flex me-3" t-ref="actions">
                 <div class="btn-group btn-group-sm">
-                    <button t-if="category.canView" class="btn btn-light" t-on-click="() => this.openCategory(category)" title="View or join channels">
+                    <button t-if="category.canView" class="btn btn-light bg-inherit border-0" t-on-click="() => this.openCategory(category)" title="View or join channels">
```

Comment:
`btn-link` instead of `border-0`?

Reply:
Again `btn-link` has the green color, and the text color also changes with :hover/:active/:focused state.

The `btn-link` has not the only property of not having border, and the other features are undesirable. It's much easier to just cancel the border.